.ci/onnx/test.sh

```diff
@@ -2,12 +2,16 @@
 
 # shellcheck source=./common.sh
 source "$(dirname "${BASH_SOURCE[0]}")/common.sh"
+source "$(dirname "${BASH_SOURCE[0]}")/../pytorch/common.sh"
 
 if [[ "$BUILD_ENVIRONMENT" == *onnx* ]]; then
   pip -q install --user "file:///var/lib/jenkins/workspace/third_party/onnx#egg=onnx"
   # TODO: This can be removed later once vision is also part of the Docker image
   pip install -q --user --no-use-pep517 "git+https://github.com/pytorch/vision.git@$(cat .github/ci_commit_pins/vision.txt)"
   # JIT C++ extensions require ninja, so put it into PATH.
   export PATH="/var/lib/jenkins/.local/bin:$PATH"
-  "$ROOT_DIR/scripts/onnx/test.sh"
+  # NB: ONNX test is fast (~15m) so it's ok to retry it few more times to avoid any flaky issue, we
+  # need to bring this to the standard PyTorch run_test eventually. The issue will be tracked in
+  # https://github.com/pytorch/pytorch/issues/98626
+  retry "$ROOT_DIR/scripts/onnx/test.sh"
```

Comment:
Is there a retry/timeout limit?

Reply:
Yup, it retries 3 times https://github.com/pytorch/pytorch/blob/master/.ci/pytorch/common_utils.sh#L12-L14 with some delay in between.  In the very very unlikely worst case where it fails 4 times in a row, it would still take about 1 hour, way less than other CI jobs.  Now if I think about this, may be retrying 2 times is good enough here.  I will tweak it accordingly

Reply:
Update to retry it only once, as I think that makes more sense for the test and doesn't cause much delay when a test actually fails